draft-nuttycom-funding-allocation.rst

```diff
@@ -171,12 +171,168 @@ complete set of funding streams will be::
+     innovation, driving sustained improvements and advancements in the
+     protocol.
+
+   - **Promotion of Decentralization**: The proposed non-direct funding model
+     stores deferred funds for future use, with the specifics of the
+     disbursement mechanism to be determined by a future ZIP. This could allow
+     the community to have a greater influence over funding decisions, aligning
+     more closely with the ethos of the Zcash project. By decentralizing the
+     allocation process, this approach has the potential to foster innovation
+     and community involvement, ensuring that development priorities are more
+     reflective of the community's needs and desires, promoting a more open,
+     transparent, and resilient ecosystem.
+
+4. **Incentives for Development and Collaboration**:
+
+   - **Creating a Strong Incentive to Implement the Disbursement Mechanism**:
+     Allocating 50% of the block reward to the lockbox indefinitely creates
+     a powerful incentive for the community to work together to implement the
+     disbursement mechanism without delay. Knowing that there is a substantial
+     amount of funds available, stakeholders will be motivated to develop and
+     agree on an effective, decentralized method for distributing these funds.
+
+   - **Incentivizing Continuous Improvements**: The accumulation of a large
+     stored fortune within the lockbox incentivizes continuous improvements
+     to the Zcash protocol and ecosystem. Developers, contributors, and
+     community members will be driven to propose and execute projects that
+     enhance the network, knowing that successful initiatives have the
+     potential to receive funding. This model fosters a culture of ongoing
+     innovation and development, ensuring that Zcash remains at the forefront
+     of blockchain technology.
+
+   - **Aligning Long-Term Interests**: By tying a significant portion of the
+     block reward to future decentralized grants funding, the model aligns
+     the long-term interests of all stakeholders. Miners, developers, and
+     community members alike have a vested interest in maintaining and
+     improving the Zcash network, as the value and success of their efforts
+     are directly linked to the availability and effective use of the lockbox
+     funds. This alignment of incentives ensures that the collective efforts
+     of the community are focused on the sustainable growth and advancement
+     of the Zcash ecosystem.
+
+Guidance on Future Requirements for Alternative 1
+'''''''''''''''''''''''''''''''''''''''''''''''''
+
+To support the motivations outlined, the following guidance is proposed for
+Alternative 1. Future ZIP(s) will define the disbursement mechanism. These
+are suggestions to achieve the outlined motivations that should be considered
+in those future ZIP(s):
+
+1. **Cap on Grants**: Grants should be capped to prevent any single project
```

Comment:
IDK if the problem is a single project using too much funds ... really it's about more granular accountability and incremental goals ... even if one project is eating up a lot of funds very fast ... that could be a good thing if they are just doing a lot of stuff that the community wants really fast ...